scripts/fetch-json.js

```diff
@@ -6,24 +6,18 @@ let chalk = require(`chalk`);
 
 shell.mkdir(`-p`, `source/json/temp/`);
 
-// Pulse
-request(`https://network-pulse-api-production.herokuapp.com/entries/?ordering=-created&page_size=996&format=json&featured=True&page=1`, (error, response, body) => {
-  if (!error && response.statusCode === 200) {
-    console.log(chalk.green(`Pulled Pulse JSON`));
-    shell.ShellString(body).to(`source/json/temp/pulse.json`);
-  } else {
-    console.log(chalk.red(`Failed to pull Pulse JSON!`));
-    shell.exit(1);
-  }
-});
+let fetch = (shortName, source) => {
```

Comment:
Can you name this something else so it isn't confusing in the future? (https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)